### PR TITLE
Alvaro Assignment 4 

### PR DESCRIPTION
**Pull request by alvarodelser**
Assignment 4 Submission. Includes Assignment 3 as well (from pull Request #110 )

### CODE REVIEW

**Comment by dgarijo** [comment]
This branch has conflicts on files that I cannot see. I can't merge it in its current state

File: Assignment4/alvarodelser-071479466/task07.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+q1 = prepareQuery('''
+  SELECT distinct ?Subject WHERE {
+    ?Subject foaf:knows ?z.
+    ?Subject foaf:knows ?y.
+    FILTER(?y != ?z)
+  }
+  ''',
+    initNs = { "foaf": FOAF}
+)
+# Visualize the results
```
Comment (dgarijo) [inline Assignment4/alvarodelser-071479466/task07.py:111]: Missing all exercises in rdflib

File: Assignment4/alvarodelser-071479466/task06.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+g.add((ns.JaneSmith, VCARD.Email, Literal("jsmith@example.org")))
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+# TO DO
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.JohnSmith, VCARD.Works, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/alvarodelser-071479466/task06.py:73]: vcard does not define `Works`. Create your own property or find another vocabulary that does

**Review by dgarijo — CHANGES_REQUESTED** [review]
Please address missing comments

**Comment by anaigmo** [comment]
Submitted and graded. The PR is conflictive and contains more files than the specified for the assignment, closing without merge

**Pull request closed without merging**